include/enmasse-kubernetes.jsonnet

```diff
@@ -10,7 +10,7 @@ local flavorConfig = import "flavor.jsonnet";
     "items": [ templateConfig.generate(use_sasl, with_kafka, false),
                addressController.deployment("enmasseproject/address-controller", "false", 0),
                restapiRoute.ingress(""),
-               addressController.service,
+               addressController.service("LoadBalancer"),
```

Comment:
As you proposed during our chat should we have a different kubernetes tenmplate for cloud provider (i.e. Azure) so that the current one using minikube can work with ingresses ?